src/style.css

```diff
@@ -54,39 +69,89 @@ body > header {
 #form > h2 {
   margin: 0;
   margin-bottom: 10px;
+  color: #38d2d2;
 }
 
 #form > input {
   margin: 15px 0;
-  border: 2px solid black;
+  align-items: center;
+  appearance: none;
+  background-image: radial-gradient(100% 100% at 100% 0, #5adaff 0, #5468ff 100%);
```

Comment:
- [OPTIONAL BUT HIGHLY RECOMMENDED] Please remove this line of code `background-image: radial-gradient(100% 100% at 100% 0, #5adaff 0, #5468ff 100%);`, in order to make your form input field to look much more intuitive and professional. :slightly_smiling_face: